### ISSUE TITLE
Squad Strike Crash

### ISSUE THREAD

**Issue by SquadStrikeHDR** [HDR-Development/HewDraw-Remix#292]
Hello! As of the beta release, Squad Strike will crash upon trying to load the second character. After testing, it appears that this will only happen with two human players. Vs. CPU appears to work normally. Here is a description of the crash:
The game plays as normal, we take the first stock, and as you probably know there's a very long time between characters in squad strike. That time passes as normal, and then approximately when the next character would appear on the angel platform (it hasn't appeared yet) that's when it crashes.
I've attached a picture of the crash screen. The error message seems to always reference the character that should have been loaded in to the game at that moment.

That's all. I wanted to thank all the devs. My friends and I are lifelong smashers, and all we ever play now is HDR, most of that time being in squad strike. This project has obviously required a lot of passion and dedication, so thank you.
![SquadStrike](https://user-images.githubusercontent.com/101666829/158450521-6aa52abd-8199-4f00-ace2-62302f40001b.jpg)

**Comment by jdipalermo27** [comment]
Does this happen with every mix of character, or just specific characters, as in is this the first occurrence or have you tried multiple times with different characters?

**Comment by SquadStrikeHDR** [comment]
I have tried 4 mixes, so 20 different characters in total. It happened the same way each time.

> Does this happen with every mix of character, or just specific characters, as in is this the first occurrence or have you tried multiple times with different characters?

**Comment by kalomaze** [comment]
On the latest beta release I have tried to crash Squad Strike like before. It appears the issue has been fixed, because after 10+ matches with various characters and stages, I haven't gotten a single Skyline panic, not even after several 5v5 matches. Jobrien has no clue why it works now. I'm assuming that this works for other people as I used to get the panic as well, so I would mark this issue as resolved
